P5/Source/Specs/encodingDesc.xml

```diff
@@ -23,14 +23,13 @@ source or sources from which it was derived.</desc>
     <memberOf key="model.teiHeaderPart"/>
   </classes>
   <content>
-    
-      
-        <alternate minOccurs="1" maxOccurs="unbounded">
-          <classRef key="model.encodingDescPart"/>
-          <classRef key="model.pLike"/>
-        </alternate>
-      
-    
+    <sequence preserveOrder="false">
+      <alternate minOccurs="1" maxOccurs="unbounded">
```

Comment:
Note that this seems to result in relaxng that does not represent the ODD's model, but it may be a separate issue.

Expected:
```xml
<interleave>
  <choice>
    <oneOrMore>
      <ref name="model.encodingDescPart"/>  
    </oneOrMore>
    <oneOrMore>
      <ref name="model.pLike"/>
    </oneOrMore>
  </choice>
  <optional>
    <ref name="constraintDecl"/>
  </optional>
</interleave>
```

Actual:
```xml
<interleave>
   <oneOrMore>
      <choice>
         <ref name="model.encodingDescPart"/>
         <ref name="model.pLike"/>
      </choice>
   </oneOrMore>
   <optional>
      <ref name="constraintDecl"/>
   </optional>
</interleave>
```